src/django_linear_migrations/management/commands/makemigrations.py

```diff
@@ -8,8 +8,15 @@
 
 
 class Command(BaseCommand):
-    def write_migration_files(self, changes: dict[str, list[Migration]]) -> None:
-        super().write_migration_files(changes)
+    def write_migration_files(
+        self,
+        changes: dict[str, list[Migration]],
+        update_previous_migration_paths: dict[str, str] | None = None,
+    ) -> None:
+        if update_previous_migration_paths:
+            super().write_migration_files(changes, update_previous_migration_paths)
+        else:
+            super().write_migration_files(changes)
```

Comment:
better to avoid the `else` clause with an explicit Django version check: `if django.VERSION >= (4, 2): ...`